### PR TITLE
fix: [FX-2401] Deleting fixed version reference to picasso-provider package

### PR DESCRIPTION
**Pull request by denieler**
[FX-2401]

### Description

Deleting fixed version reference to picasso-provider package.

P.S.
Changesets now reporting this message:

Screenshot 2022-01-03 at 21 07 39

Ideally, we should remove it somehow.

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-2401]: https://toptal-core.atlassian.net/browse/FX-2401?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by TomasSlama — APPROVED** [review]
why we don't use `^0.5.0`?

**Comment by denieler** [comment]
> why we don't use ^0.5.0?

@TomasSlama to let users who have dependency on `0.4.0` Picasso-Provider right now to keep it and not upgrade Picasso-Lab

File: yarn.lock [hunk 1]
```diff
@@ -4898,6 +4898,17 @@
     micromatch "^4.0.4"
     requireindex "^1.2.0"
 
+"@toptal/picasso-provider@^0.4.0":
+  version "0.4.1"
```
Comment (denieler) [inline yarn.lock:4902]: this dependency will be taken from the npm registry, not from the monorepo, but probably it's ok

File: packages/picasso-lab/src/Calendar/Calendar.tsx [hunk 2]
```diff
@@ -26,7 +26,7 @@ type SimpleReactCalendarRangeType = {
 export type DateOrDateRangeType = Date | DateRangeType
 export type DateRangeType = [Date, Date]
 
-export { DayProps }
+export type { DayProps }
```
Comment (denieler) [inline packages/picasso-lab/src/Calendar/Calendar.tsx:29]: to fix `yarn build:storybook` command warnings

Screenshot 2022-01-04 at 10 56 00

File: packages/picasso/src/Container/Container.tsx [hunk 3]
```diff
@@ -170,4 +170,4 @@ Container.defaultProps = {
 }
 
 export default Container
-export { VariantType }
+export type { VariantType }
```
Comment (denieler) [inline packages/picasso/src/Container/Container.tsx:173]: see https://github.com/toptal/picasso/pull/2362/files#r777915894

**Review by OndrejTuma — APPROVED** [review]
👍

**Pull request merged**